src/currencyRatesFileGen.js

```diff
@@ -0,0 +1,59 @@
+// prebidCurrencyRatesFileGenerator
+// loads currency rates from api.fixer.io and creates a JSON representation in S3
+// revision:
+var http = require('http');
+var AWS = require('aws-sdk');
+
+var fromCurrencies = ['USD','GBP'];
+var supportedCurrencies = 'AUD,BRL,CAD,CHF,CNY,CZK,DKK,EUR,GBP,HKD,HUF,IDR,ILS,INR,JPY,KRW,MXN,MYR,NOK,NZD,PHP,PLN,RUB,SEK,SGD,THB,TRY,USD,ZAR';
```

Comment:
could use `const` here if the node version supports it. Probably worth adding the target node version in the package.json